tests/nominal/test_BaseNominalTransformer.py

```diff
@@ -29,7 +29,7 @@ def test_not_fitted_error_raised(self, initialized_transformers):
             with pytest.raises(NotFittedError):
```

Comment:
Q: will be a separate issue, but should we create a FitTestsMixin class to cover this one?

Reply:
Arguably.  Would only be inherited here an in test_BaseImputer so might actually add more complexity right now.  honestly I feel like this might be best left for now but happy to raise an issue or make a quick change if you think it would be better.

Reply:
set up an issue so can look into fully at later date! https://github.com/lvgig/tubular/issues/264